spec/core/v2/ics-004-packet-semantics/PACKET.md

```diff
@@ -0,0 +1,77 @@
+# Packet Specification
+
+## Packet V2
+
+The IBC packet sends application data from a source chain to a destination chain with a timeout that specifies when the packet is no longer valid. The packet will be committed to by the source chain as specified in the ICS-24 specification. The receiver chain will then verify the packet commitment under the ICS-24 specified packet commitment path. If the proof succeeds, the IBC handler sends the application data(s) to the relevant application(s).
+
+```typescript
+interface Packet {
+    sourceIdentifier: bytes,
+    destIdentifier: bytes,
+    sequence: uint64
+    timeout: uint64,
+    data: [Payload]
+}
+
+interface Payload {
+    sourcePort: bytes,
+    destPort: bytes,
+    version: string,
```

Comment:
Later on we refer to this field as AppVersion. Should we call this field appVersion to avoid misunderstanding with the protocol version?

Reply:
commit `c7b2e6d5184b5310843719b428923e0c5ee5a026` avoid misunderstandings.